docs/content/design/coordinator.md

```diff
@@ -241,6 +276,11 @@ Returns total size and count for each interval within given isointerval.
 
 Returns total size and count for each datasource for each interval within given isointerval.
 
+#### Compaction Configs
+
+* `/druid/coordinator/v1/config/{dataSource}`
```

Comment:
is this endpoint specific to compaction config or is it meant to support any dataSource level dynamic config?

Reply:
It's specific to compactionConfig. I changed the endpoint to ```/druid/coordinator/v1/config/compaction/{dataSource}```.